### PR TITLE
Hendrik wb

### PR DESCRIPTION
**Pull request by HendrikVE**
updating master for deployment on server. includes small changes, mainly on populate script and a new gender option

### CODE REVIEW

File: django/api/views.py [hunk 1]
```diff
@@ -27,9 +27,10 @@
 from rest_framework.exceptions import ValidationError
 from django.db import IntegrityError
 
+
 class NestedMultipartParser(parsers.MultiPartParser):
     def parse(self, stream, media_type=None, parser_context=None):
-        result = super().parse(stream=stream, media_type=media_type, parser_context=parser_context)
+        result = super(NestedMultipartParser, self).parse(stream=stream, media_type=media_type, parser_context=parser_context)
```
Comment (jia200x) [inline django/api/views.py]: why is this change here?

File: django/api/views.py [hunk 2]
```diff
@@ -61,8 +63,32 @@ def get_queryset(self):
                 queryset = queryset.exclude(name=app.name)
         return queryset
 
-    #TODO: Add auth
-    @detail_route(methods=['get'])
```
Comment (jia200x) [inline django/api/views.py]: This is already handled by viewsets.ModelViewSet. Please, drop this function

File: django/api/models.py [hunk 3]
```diff
@@ -21,6 +21,7 @@ class UserProfile(models.Model):
     GENDER_CHOICES = (
         ('M', 'Male'),
         ('F', 'Female'),
+        ('U', 'Undefined'),
```
Comment (jia200x) [inline django/api/models.py]: This comment is not blocking, But, what's the reason of this change?

Comment (jia200x) [reply django/api/models.py]: Check https://www.formisimo.com/blog/improve-user-experience-with-gender-neutral-options-in-webforms/

Comment (HendrikVE) [reply django/api/models.py]: This option is for people who dont want to share their gender with everyone in the internet. Im not sure if we need it at all though

File: django/api/views.py (hunk 1 above)
Comment (HendrikVE) [reply django/api/views.py]: Oh that was just a driveby change for python compatibility, but does not make sense in case of preconfigured docker container, so I drop this

File: django/api/models.py (hunk 3 above)
Comment (danpetry) [reply django/api/models.py]: Third option could be "gender diverse"?

Comment (danpetry) [reply django/api/models.py]: https://www.quora.com/How-should-I-ask-about-gender-on-a-survey-in-a-non-binary-way

Comment (danpetry) [reply django/api/models.py]: Could also provide a text input form...

**Comment by jia200x** [comment]
all good. ACK
